compose-examples/teku-only/docker-compose.yaml

```diff
@@ -79,6 +79,24 @@ services:
     volumes:
       - ./config/grafana/provisioning-t:/etc/grafana/provisioning:ro
       - ./data/grafana:/var/lib/grafana
+    environment:
+      GF_RENDERING_SERVER_URL: http://renderer:8082/render
+      GF_RENDERING_CALLBACK_URL: http://grafana:3000    
+    <<: *logging
+
+  renderer:
```

Comment:
Can we run this service with `user: "2000"` as well?

Reply:
Sure, now it's running with `user: 2000`